### PR TITLE
Handy things for Heroku

### PR DESCRIPTION
**Pull request by danielmorrison**
- gzip (cedar stack doesn't do this for us)
- rails_12factor gem

### CODE REVIEW

File: config.ru [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # This file is used by Rack-based servers to start the application.
 
 require ::File.expand_path('../config/environment',  __FILE__)
+use Rack::Deflater
```
Comment (laserlemon) [inline config.ru]: Seems like awkward placement. What about [this](http://robots.thoughtbot.com/content-compression-with-rack-deflater#rails-3-and-4)?

Comment (danielmorrison) [inline config.ru]: Well, my method is only one line! :wink: 

I guess I don't really care. I just always do it this way.

**Comment by laserlemon** [comment]
:vertical_traffic_light: :bomb: on the streaming endpoint tests. Restarting…

**Comment by laserlemon** [comment]
:vertical_traffic_light: :bomb: :boom: again. Could it be an issue with streaming GZipped content?

**Comment by laserlemon** [comment]
Same failures locally. :frowning:

**Comment by laserlemon** [comment]
:eyes: :pray: It's a bit hacky.

**Pull request merged**